arch/arm64/include/a64/chip.h

```diff
@@ -39,21 +39,24 @@
 
 /* Allwinner A64 Generic Interrupt Controller v2: Distributor and Redist */
 
-#define CONFIG_GICD_BASE          0x1C81000
-#define CONFIG_GICR_BASE          0x1C82000
+#define CONFIG_GICD_BASE          0x8000000
+#define CONFIG_GICR_BASE          0x80a0000
+#define CONFIG_GICR_OFFSET        0x20000
```

Comment:
According to the [Linux Device Tree for RK3399](https://github.com/torvalds/linux/blob/master/arch/arm64/boot/dts/rockchip/rk3399.dtsi#L549-L553):
- GICD = 0xfee00000
- GICR = 0xfef00000

But I'm not sure what's CONFIG_GICR_OFFSET. Maybe we stick with the current value and test whether UART Interrupts are OK?